lib/osf-components/addon/components/registries/finalize-registration-modal/component.ts

```diff
@@ -38,6 +41,18 @@ export default class FinalizeRegisrationModalComponent extends Component {
         }
     }
 
+    @computed('manager.draftManager.provider')
+    get noticeText() {
+        const translationOptions = { learnMoreLink: this.learnMoreLink, htmlSafe: true };
+        if (this.manager.draftManager.provider.reviewsWorkflow) {
```

Comment:
Not necessarily suggesting a change, but is there a compelling reason to do this logic in the component rather than the template?

Reply:
I've opted to add this logic in the component file since there will be more logic added as we add conditions to check if this is a no-project registration, and didn't want the template to get too messy (would have to repeat the options for this translation multiple times).

Reply:
Does it make sense then to do this logic in `draftManager`?

Reply:
I think you're right, that a computed property on the `draftManager` would be easier to use. I think it will work better for the no-project conditions as well